articles/security/oauth2-bcp.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Current Best Practices in OAuth2
+toc: true
+crews: crew-security
+topics:
+  - security
+  - OAuth2
+  - Best Practice
+  - users
+contentType:
+  - reference
+useCase:
+  - development
+---
+
+# OAuth2 Current Best Practice
+
+We have compiled several security guidelines for the different grant types specified in the OAuth RFC.
+
+Please be aware that this is not an exhaustive list; further readings and references may be found in the Keep reading section.
+
+## Common Guidelines
+
+* `redirect_uri` should be as strict as possible; http urls should not be allowed in a production application.
```

Comment:
I didn't mention that because it's common practice allowing http for development purposes. 
Security isn't the main objective in development environments. 

As for schemes, it can differ from webapps to native apps, and can get a little complicated when discussing in detail. I'd like to keep this simple, but if you have a good reference I can link them here.